### ISSUE TITLE
Whitelist Remote Images From Specific Email Addresses/Domains

### ISSUE THREAD

**Issue by seniorm0ment** [thunderbird/thunderbird-android#7254]
An option to whitelist a specific email address, or domain, to always show remote images by default would be amazing.
Block anything new by default still, unless from that address or domain.

**Comment by Cris--** [comment]
Yes please! Simple and more usebility karma for Thunderbird mobile.

**Comment by kewisch** [comment]
Thanks so much for sharing your idea. For enhancement suggestions like this, would you mind posting them over on [Mozilla Connect](https://connect.mozilla.org/t5/ideas/idb-p/ideas/label-name/thunderbird%20android)? This is a great place for the community to discuss and help shape new ideas together, and is something we review regularly to determine the future roadmap.

This would be a great fit because it provides some similarity to Thunderbird Desktop where you can also allow-list emails and domains, though I'd like to get more sentiment on how useful community members would see this feature.

To get the most out of your post, check out [this quick guide](https://connect.mozilla.org/t5/discussions/how-to-submit-a-great-idea-in-five-easy-steps/m-p/24) on crafting impactful suggestions. Focusing on the problem your idea solves and how it could help others can really make it stand out!

I’ll be closing this issue to keep our GitHub focused on active development, but I hope to see your idea over there.
